frontend/src/pages/EventSignInPage/index.tsx

```diff
@@ -65,16 +65,21 @@ function EventSignInPage(): JSX.Element {
 
               <Grid item>
                 <Grid component='div'>
-                  <Typography className={classes.eventName}
+                  <Typography
+                    className={classes.eventName}
                     variant='subtitle1'
-                    align='center'>
-                    Hosts: {event.hosts
```

Comment:
Just curious, but I remember you telling me in my pull request that...

`Hosts:{' '}
{event.hosts
    .map(host => `${host.firstName} ${host.lastName}`)
    .join(', ')}
</Typography>`

should be changed to...

`Hosts: {event.hosts
    .map(host => `${host.firstName} ${host.lastName}`)
    .join(', ')}`

However, it looks like you have reverted that change. Did you change your mind about the formatting?

Reply:
This is only because I made this branch before your changes were merged into master rercently, so I don't have the updates from master yet. You'll see when you scroll down to the end of this page that this branch is outdated. I was going to update it with changes from master once I get the necessary approvals. So I didn't change my mind about the formatting

Reply:
I see, thanks for the clarification!